### PR TITLE
German Translation improved

### PR DESCRIPTION
**Pull request by nbessi**
Replace #70

### CODE REVIEW

**Comment by nbessi** [comment]
waiting for https://github.com/OCA/l10n-switzerland/pull/101

**Comment by yvaucher** [comment]
#101 merged

**Comment by yvaucher** [comment]
travis restarted

**Comment by yvaucher** [comment]
:+1:

**Comment by nbessi** [comment]
One more thumb please this is just translations :tongue:

File: l10n_ch_payment_slip/i18n/de.po [hunk 1]
```diff
@@ -57,13 +57,13 @@ msgstr "Horiz. position in mm für die Adresse"
 #: model:ir.model.fields,field_description:l10n_ch_payment_slip.field_res_company_bvr_background
 #: field:res.company,bvr_background:0
 msgid "Insert BVR background ?"
-msgstr "BVR Hintergrund einfügen ?"
+msgstr "ESR Hintergrund einfügen ?"
```
Comment (eleu) [inline l10n_ch_payment_slip/i18n/de.po]: msgstr "ESR Hintergrund einfügen?"

(normally, there is no space before a question mark / exclamation mark)

File: l10n_ch_payment_slip/i18n/de.po [hunk 2]
```diff
@@ -74,19 +74,19 @@ msgstr "Import"
...
 msgid "BVR vert. position for address (mm)"
-msgstr "BVR vert. position für die Adresse (mm)"
+msgstr "ESR vertikale Position der Adresse in mm"
 
 #. module: l10n_ch_payment_slip
 #: model:ir.model.fields,field_description:l10n_ch_payment_slip.field_res_partner_bank_print_bank
 #: field:res.partner.bank,print_bank:0
 msgid "Print Bank on BVR"
-msgstr "Bank Daten auf dem BVR drucken"
+msgstr "Bank Daten auf dem ESR drucken"
```
Comment (eleu) [inline l10n_ch_payment_slip/i18n/de.po]: msgstr "Bank Daten auf den ESR drucken"

File: l10n_ch_payment_slip/i18n/de.po [hunk 3]
```diff
@@ -98,30 +98,30 @@ msgstr "BVR Import"
...
 msgid "Journal Items"
-msgstr ""
+msgstr "Journal-Einträge"
 
 #. module: l10n_ch_payment_slip
 #: model:ir.model.fields,field_description:l10n_ch_payment_slip.field_res_partner_bank_print_account
 #: field:res.partner.bank,print_account:0
 msgid "Print Account Number on BVR"
-msgstr "Konto Nummer auf dem BVR drucken"
+msgstr "Konto Nummer auf dem ESR drucken"
```
Comment (eleu) [inline l10n_ch_payment_slip/i18n/de.po]: msgstr "Kontonummer auf den ESR drucken"

**Comment by yvaucher** [comment]
First commit authoring must be fixed if possible
https://github.com/nbessi/l10n-switzerland/commit/c8085500e2b8c4235b9269e7b69833f5879225ff
There is no mail set.

**Comment by nbessi** [comment]
@yvaucher @gurneyalex is there some procedure to do so somewhere. 
Else if it is not problematic with OCA, CLA, we commit as it.

**Comment by yvaucher** [comment]
Ok my mistake there is a mail on this commit.

@michi it seems you github account is not properly settup you should add you email to it.

**Comment by yvaucher** [comment]
@nbessi I think we can merge then once @eleu comments are addressed

**Comment by nbessi** [comment]
@yvaucher I have pushed the fixed but I'm bothered by the CLA runbot. Can someone give me some clues about Michael Volz please

**Comment by yvaucher** [comment]
@nbessi As it is for translations I think we can merge it anyway. No clue about @michl mail address.

**Pull request merged**